OpenRA.Mods.Common/Traits/World/ExperimentalMapGenerator.cs

```diff
@@ -1614,68 +1616,68 @@ void TestSymmetry(CPos[] sources, CPos destination)
 					}
 
 					// Closer to +inf means "more preferable" for plan.
-					var plan1024ths = new CellLayer<int>(map);
+					var plan1048576ths = new CellLayer<int>(map);
```

Comment:
I'm not convinced that `1048576ths` vs `1024ths` is actually helpful anymore for humans - these are basically just arbitrary numbers now. How about renaming to `planScores`/`planPriorities`/etc which IMO would be clearer?

Reply:
I've renamed `plan1048576ths` to just `plan`. The scale label is somewhat less meaningful at this stage.

I've kept `priorities` as is (that doesn't have the 1024ths vs 1048576ths labelling anyway).

Avoiding other renaming as it's not generally related to this change.